src/sorting/sorting.py

```diff
@@ -1,28 +1,110 @@
 # TO-DO: complete the helper function below to merge 2 sorted arrays
 def merge(arrA, arrB):
     elements = len(arrA) + len(arrB)
+    # creates an aray of 0s that is len(elements)
     merged_arr = [0] * elements
+    # merged_arr = [0 for _ in range(elements)] # list comprehension version to accomplish line above
 
     # Your code here
+    a = 0
+    b = 0
+    # [38, 44] [3, 5]
 
+    for i in range(elements):
+        # check if a is in bounds of array a
+        if a >= len(arrA):
+            # we've moved all the elements from aarA into mergedArr
+            # we still have elements from arrB that need to be moved
```

Comment:
Nice, nice, nice. I like the notes here. I'm glad we were able to make some sense of what's going on here last night.